mycqu/card.py

```diff
@@ -178,7 +164,7 @@ def from_dict(data: dict[str, Any]) -> Bill:
         """
         return Bill(
             tran_name=data['tranName'],
-            tran_date=data['tranDt'],
+            tran_date=datetime.datetime.strptime(data['tranDt'], "%Y-%m-%d %H:%M:%S"),
```

Comment:
直接 `strptime` 得到的时间也是会按照运行环境的时区，可以用 `replace` 将北京时间应用上去：
```suggestion
            tran_date=datetime.datetime.strptime(data['tranDt'], "%Y-%m-%d %H:%M:%S").replace(tzinfo=TIMEZONE),
```
不过全堆在这行似乎长了点，也可以换种写法…